src/components/utils/DEDialog.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * A general purpose dialog
+ *
+ * @author sriram
+ *
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+
+import { build } from "@cyverse-de/ui-lib";
+
+import ids from "./ids";
+
+import {
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    IconButton,
+    useMediaQuery,
+    useTheme,
+} from "@material-ui/core";
+
+import CloseIcon from "@material-ui/icons/Close";
+
+function DEDialog(props) {
+    const { baseId, open, title, onClose, content, actions } = props;
+    const { t } = useTranslation("common");
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down("sm"));
+    const dialogTitleId = build(baseId, ids.DIALOG.TITLE);
+    return (
+        <Dialog
+            open={open}
+            onClose={onClose}
+            maxWidth="sm"
+            fullWidth
+            fullScreen={fullScreen}
+            aria-labelledby={dialogTitleId}
+        >
+            <DialogTitle>
```

Comment:
I see `aria-labelledby` being set in the `Dialog`, but I think this `DialogTitle` is then missing the `dialogTitleId` being set